drivers/spi/spi_ll_stm32.c

```diff
@@ -846,6 +846,11 @@ static int spi_stm32_init(const struct device *dev)
 		return err;
 	}
 
+	err = spi_context_cs_configure_all(&data->ctx);
```

Comment:
If this logic makes sense generically, the rest of the SPI drivers should also be updated to do this.
If it does not make sense generally, the new functions probably shouldn't be in the global headers.
I don't think we want to introduce more family specific behaviors if we can help it (SPI CS lines are setup on STM, but not any others).

Reply:
The aim is to provide the same support for all supported drivers but currently I have only one device with stm32f4 that can be used for testing. Next week, I'll get some devboards with nRF52 chips so I'll add these changes for them too.